app/main/views/suppliers.py

```diff
@@ -77,7 +77,24 @@ def dashboard():
     ), 200
 
 
-@main.route('/edit', methods=['GET'])
```

Comment:
Oh I just noticed in the Trello card it says "302 redirect existing edit page to new edit page"... do we care about doing this?  

I can't imagine anyone has the existing edit page bookmarked...